tests/numpy/test_numpy_wrapper.py

```diff
@@ -31,7 +31,7 @@ class TestExtractTensors:
     def test_empty_terable(self):
         """Test that an empty iterable returns nothing"""
         res = list(np.extract_tensors([]))
-        assert res == []
+        assert not res

```

Comment:
```suggestion
        assert res == [] # pylint: disable=use-implicit-booleaness-not-comparison
```
but I might have spelled booleaness wrong.

Basically, asserting `res` is falsy is not the same as assert that it's an empty list.  The output could have been `False` for some reason, and we would need to catch that.

Reply:
generally I agree, but the line right above it says `res = list(...)`, so we know it's a list object. the only way a list object is falsy is if it's empty